addons/account/wizard/account_payment_register.py

```diff
@@ -411,13 +411,16 @@ def _compute_communication(self):
                 wizard.communication = False
 
     @api.depends('can_edit_wizard')
+    @api.depends_context('force_group_payment')
     def _compute_group_payment(self):
-        for wizard in self:
-            if wizard.can_edit_wizard:
-                batches = wizard._get_batches()
-                wizard.group_payment = len(batches[0]['lines'].move_id) == 1
-            else:
-                wizard.group_payment = False
+        self.group_payment = self._context.get('force_group_payment')
+        if not self.group_payment:
+            for wizard in self:
+                if wizard.can_edit_wizard:
+                    batches = wizard._get_batches()
+                    wizard.group_payment = len(batches[0]['lines'].move_id) == 1
+                else:
+                    wizard.group_payment = False
```

Comment:
```suggestion
        force_group_payment = self._context.get('force_group_payment')
        for wizard in self:
            if force_group_payment: 
                wizard.group_payment = True
            elif wizard.can_edit_wizard:
                batches = wizard._get_batches()
                wizard.group_payment = len(batches[0]['lines'].move_id) == 1
            else:
                wizard.group_payment = False
```

That should make the branch green ;)